alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/destination/Destination.kt

```diff
@@ -187,10 +208,12 @@ data class Destination(
                 id = sin.readString(),
                 version = sin.readLong(),
                 schemaVersion = sin.readInt(),
+                seqNo = sin.readInt(),
+                primaryTerm = sin.readInt(),
                 type = sin.readEnum(DestinationType::class.java),
                 name = sin.readString(),
                 user = if (sin.readBoolean()) {
-                    User(sin)
+                    User.readFrom(sin)
```

Comment:
Looks like you over wrote my change ;) . `User(sin)`

Reply:
When I cherry picked changes, it looked like it was changed to `User.readFrom(sin)`. Just changed it back